python/ray/serve/llm/__init__.py

```diff
@@ -1,5 +1,8 @@
 from typing import TYPE_CHECKING, Any, Dict, Optional, Union
 
+# Using Deprecated copied from ray.rllib.utils.deprecation since they are returning better messages.
+# TODO: Ray core should inherit that.
```

Comment:
nit: We usually put the name here, so we can map back who wrote the todo (e.g. `TODO (ahao)`)